explorer/client/src/reducers/adminHeads.ts

```diff
@@ -3,20 +3,41 @@ import { Reducer } from 'redux'
 import { Head } from 'explorer/models'
 
 export interface State {
-  items?: Record<string, Head>
+  loading: boolean
+  error: boolean
+  items?: Record<Head['id'], Head>
 }
 
-const INITIAL_STATE: State = {}
+const INITIAL_STATE: State = {
+  loading: false,
+  error: false,
+}
 
 export const adminHeads: Reducer<State, Actions> = (
   state = INITIAL_STATE,
   action,
 ) => {
   switch (action.type) {
+    case 'FETCH_ADMIN_HEADS_BEGIN':
+      return {
+        ...state,
+        loading: true,
+      }
```

Comment:
What do you think about adding `error: false` here? 
So each time we begin a fetch we restart the error state.